components/elm/bld/namelist_files/namelist_definition.xml

```diff
@@ -309,6 +309,17 @@ Allowed values are:
  5 : use gross domestic production and population datasets to simulate anthropogenic fire supression
 </entry>
 
+<entry id="fates_harvest_mode" type="integer" category="physics"
+        group="elm_inparm" valid_values="0,1,2,3,4", value="0" >
+Set FATES harvesting mode by setting fates_harvest_mode > 0.
+Allowed values are:
+ 0 : no fates harvesting of any kind
```

Comment:
I like how you basically use fates_harvest_mode as an enum in the code, but might it be better if the user-facing interface here used strings?

Reply:
I think I agree with this. I for one will have to look up what the integers are every time. So maybe better to use strings instead. Perhaps `no_harvest`=0, `event_code`=1, `surfdata_file`=2, `luh_area_based`=3, `luh_mass_based`=4?